### ISSUE TITLE
Warning: color.c: In function 'freerdp_image_copy_from_pointer_data':

### ISSUE THREAD

**Issue by CHERTS** [FreeRDP/FreeRDP#3236]
When build (branch master) Warning message:

``````
[ 67%] Building C object libfreerdp/CMakeFiles/freerdp.dir/codec/color.c.o
/ts/ports/opt/freerdp/work/src/FreeRDP/libfreerdp/codec/color.c: In function 'freerdp_image_copy_from_pointer_data':
/ts/ports/opt/freerdp/work/src/FreeRDP/libfreerdp/codec/color.c:1620:37: warning: 'andBits' may be used uninitialized in this function [-Wmaybe-uninitialized]
       if (!(andBit >>= 1)) { andBits++; andBit = 0x80; }

```                                ^

Fix it, please.
``````

**Comment by bmiklautz** [comment]
@CHERTS pull requests are welcome!

**Comment by bmiklautz** [comment]
@CHERTS please add some additional infomation:
* system compiled on
* compiler and compiler version used
* cmake configuration

**Comment by CHERTS** [comment]
I compile freerdp from thinstation (master branch)
cmake version 3.2.2
gcc (Thinstation) 4.8.4
>Using built-in specs.
>COLLECT_GCC=gcc
>COLLECT_LTO_WRAPPER=/usr/lib/gcc/i686-pc-linux-gnu/4.8.4/lto-wrapper
>Target: i686-pc-linux-gnu
>Configured with: ../gcc-4.8.4/configure --prefix=/usr --mandir=/usr/man --libexecdir=/usr/lib --enable-languages=c,c++,objc --enable-threads=posix --enable-__cxa_atexit --enable-clocale=gnu --enable-shared --disable-nls --with-x=no --with-system-zlib --enable-multilib --with-pkgversion=Thinstation
>Thread model: posix
>gcc version 4.8.4 (Thinstation)

**Comment by akallabeth** [comment]
Fixed in current master 3c7cc555a5a5f02366918be7eb935bb1f22115ef
